CONTRIBUTING

```diff
@@ -0,0 +1,118 @@
+
+Once you see an issue that you'd like to work on, please post a comment saying that you want to work on it. Something like "I want to work on this" is fine.
+
+### Submit Pull Requests
+
+Pull Requests are always welcome, even for small fixes like typos. Check [What to Contribute](#what-to-contribute) above for more information about what kind of PRs we are looking for.
+
+#### Pull Request Checklist
+
+When you submit your pull request, or you push new commits to it, our automated systems will run some checks on your new contribution. We require that your pull request passes these checks, but we also have more criteria than just that before we can accept and merge it. We recommend that you check the following things locally before you submit your change:
+
+- We use GitHub Actions to test all pull requests. We prefer that all tests succeed on a pull request before it is merged. Our repositories also contain a *Makefile* for running linting tasks locally. Executing `make lint` is another way to check your work before GitHub actions.
+
+### Review/Comment on Pull Requests
+
+Reviews and comments on open Pull Requests are always appreciated.  A minimum of 3 community approvals, from at least 2 different companies, are required to merge PRs in any of our repositories -  so your review is greatly appreciated.
+
+Where a comment is better expressed as a proposed change, the change can be made directly either by editing the file (right hand "..." -> "edit file" for larger changes, or the "document-with-+-" icon above the comment window for smaller ones).   The author can accept these changes directly, which is much easier for them than writing a change themselves.
+
+#### Steps for Creating a New PR
+
+- Once you have made your change or added new content
+- Ensure you are up to date with the `main` branch of the cnf-wg repository
+- Open a new Pull Request in the repository of the correct Focus Area: i.e. [Best Practices](https://github.com/lfn-cnti/bestpractices/pulls), [Test Catalog](https://github.com/cnti-testcatalog/testsuite/pulls), or [Certification](https://github.com/lfn-cnti/certification/pulls)
+- Choose at least 3 reviewers OR choose a [TSC member](https://wiki.lfnetworking.org/display/LN/CNTi+Governance) who will help to find reviewers for you
+
+#### Steps to Accept a PR
+
+1. Reviewers will review the PR and give their feedback: approve, request changes, or comment w/o approval
+1. After the required number of approvals from reviewers is reached the PR may be merged
+
+Anyone with merge access can merge the PR after the item has been approved.
+
+### Acceptance Criteria for Contributions
+
+Reviewers - Any community member
```

Comment:
Any contributor of respective repository

Reply:
Agreed.  New commit made.